scripts/json_overview_image_info.py

```diff
@@ -14,23 +14,37 @@
 
 assert getenv("WORK_DIR"), "$WORK_DIR required"
 
-work_dir = Path(getenv("WORK_DIR"))
-
-output = {}
-
-for json_file in work_dir.glob("*.json"):
-    image_info = json.loads(json_file.read_text())
-    if not output:
-        output.update(image_info)
-    else:
-        # get first (and only) profile in json file
-        device_id = next(iter(image_info["profiles"].keys()))
-        if device_id not in output["profiles"]:
-            output["profiles"].update(image_info["profiles"])
-        else:
-            output["profiles"][device_id]["images"].append(
-                image_info["profiles"][device_id]["images"][0]
-            )
+# merge profiles
+def get_output(work_dir):
+    output = None
+
+    # preserve existing profiles.json file data
+    if output_path.is_file():
+        output = json.loads(output_path.read_text())
```

Comment:
@aparcar we still need this for the image builder.

Reply:
This needs to be limited to imagebuilder only or requires a smarter logic to understand changes of version.